server/test/src/main/java/org/eclipse/lsp/cobol/test/engine/TestData.java

```diff
@@ -33,15 +35,35 @@ public class TestData {
   Map<String, List<Diagnostic>> diagnostics;
   Map<String, List<Location>> variableDefinitions;
   Map<String, List<Location>> variableUsages;
-  Map<String, List<Location>> paragraphDefinitions;
-  Map<String, List<Location>> paragraphUsages;
-  Map<String, List<Location>> sectionDefinitions;
-  Map<String, List<Location>> sectionUsages;
+  Map<ProcedureId, List<Location>> procedureDefinitions;
+  Map<ProcedureId, List<Location>> procedureUsages;
```

Comment:
Are we loosing data here? In the following snipper
```
                PERFORM P OF S.
```
I would expect go to definition on `P` to go to the paragraph `P` in section `S`, and go to def on `S` to just go to the statement that introduces the section `S`.
And therefore I'd expect the appropriate annotations in tests as well.

Reply:
We didn't have place to put the data before. There is a bug related to it, currently we resolve usage reference to all paragraphs with the name P not only in tests but in the analysis results. So I plan couple more iterations to resolve it.